### PR TITLE
fix(gum) add event handling for SLOW_GET_USER_MEDIA

### PR DESCRIPTION
**Pull request by quitrk**
Show an overlay with a spinner when slow gUM is fired

### CODE REVIEW

File: react/features/overlay/components/web/SlowGUMOverlay.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import Spinner from '@atlaskit/spinner';
+import React from 'react';
+
+import { translate } from '../../../base/i18n';
+
+import AbstractSlowGUMOverlay from './AbstractSlowGUMOverlay';
+import OverlayFrame from './OverlayFrame';
+
+/**
+ * Implements a React Component for suspended overlay. Shown when a suspend is
```
Comment (saghul) [inline react/features/overlay/components/web/SlowGUMOverlay.js:11]: please update the docs

File: conference.js [hunk 2]
```diff
@@ -499,6 +499,11 @@ export default {
                     mediaPermissionPromptVisibilityChanged(true, browserName))
         );
 
+        JitsiMeetJS.mediaDevices.addEventListener(
+            JitsiMediaDevicesEvents.SLOW_GET_USER_MEDIA,
```
Comment (paweldomas) [inline conference.js:503]: there's no such event `JitsiMediaDevicesEvents.SLOW_GET_USER_MEDIA` right now. Is there a PR for lib-jitsi-meet? https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiMediaDevicesEvents.js

Comment (quitrk) [reply conference.js:503]: https://github.com/jitsi/lib-jitsi-meet/pull/1406

**Review by paweldomas — DISMISSED** [review]
LGTM, but please check with @saghul before merging

**Review by saghul — DISMISSED** [review]

**Review by saghul — APPROVED** [review]

**Review by muscat1 — APPROVED** [review]

**Pull request merged**